engine_test.go

```diff
@@ -1755,16 +1755,47 @@ func TestOrderByColumns(t *testing.T) {
 }
 
 func TestInsertInto(t *testing.T) {
-	e := newEngine(t)
-	testQuery(t, e,
-		"INSERT INTO mytable (s, i) VALUES ('x', 999);",
-		[]sql.Row{{int64(1)}},
-	)
+	var insertions = []struct {
+		insertQuery    string
+		expectedInsert []sql.Row
+		selectQuery    string
+		expectedSelect []sql.Row
+	}{
+		{
+			"INSERT INTO mytable (s, i) VALUES ('x', 999);",
```

Comment:
You should have tests of all the supported data types. I found bugs with handling floats in the past.

Also test inserting literal NULL values.